<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GS1 Flash/Unreal Text/Code Generator</string>
  </property>
  <property name="locale">
   <locale language="Persian" country="Iran"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tblMessages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>701</width>
      <height>371</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>390</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save All Changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRevert">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Revert All Changes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerate">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Text/Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEraseRow">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase Row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddRow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Row</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
